src/quantum/README.rst

```diff
@@ -0,0 +1,161 @@
+Microsoft Azure CLI 'quantum' Extension
+==========================================
+
+Azure Quantum is the first open Quantum computing platform. It offers a range of services
+from quantum hardware to full-state simulators and quantum inspired optimizations,
+providing developers and customers access to the most competitive quantum offering
+on the market.
+
+To learn more about azure quantum visit:
+https://azure.microsoft.com/en-us/services/quantum/
+
+To learn more about quantum computing and Microsoft's Quantum Development Kit visit:
+https://docs.microsoft.com/quantum/
+
+
+# Using the `az quantum` extension to list and manage jobs in Azure Quantum
+
+1. Log in to Azure using your credentials.
+   ```dotnetcli
+   az login
+   ```
+>[!NOTE] In case you have more than one subscription associated with your Azure account you must specify the 
+>subscription you want to use. You can do this with the command `az account set -s <Your subscription ID>`.
+
+2. You can see all the Azure Quantum workspaces in your subscription with the `az quantum workspace list` command:
+   ```dotnetcli
+   az quantum workspace list
+   ```
+
+3. You can use `quantum workspace set` to select a default workspace you want to use 
+   to list and submit jobs. Note that you also need to specify the resource group.
+   ```dotnetcli
+    az quantum workspace set -g MyResourceGroup -w MyWorkspace  -o table
+
+    Location     Name                               ResourceGroup
+    -----------  ---------------------------------  --------------------------------
+    westus       ws-yyyyyy                          rg-yyyyyyyyy
+   ```
+
+4. You can check the current default workspace with the command `az quantum workspace show`.
+
+   ```dotnetcli
+    az quantum workspace show -o table
+
+    Location     Name                               ResourceGroup
+    -----------  ---------------------------------  --------------------------------
+    westus       ws-yyyyyy                          rg-yyyyyyyyy
+   ```
+
+5. You can see all the jobs submitted a workspace using `az quantum job list`.
+
+   ```dotnetcli
+    az quantum job list -o table
+
+    Id                                    State    Target          Submission time
+    ------------------------------------  -------  --------------  ---------------------------------
+    yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyy  Waiting  MyProvider.MyTarget  2020-06-12T14:20:18.6109317+00:00
+   ```
+   The console will output the information about the job, including the ID of the job.
+
+6. You can use the ID of the job to track its status:
```

Comment:
You use "." at the end of the sentences in previous items.

Reply:
Fixed.